x/launch/types/genesis_url.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+)
+
+const HashLength = 64
+
+// NewGenesisURL returns a genesis URL from the URL
+func NewGenesisURL(url string, hash string) (gu GenesisURL, err error) {
+	if len(url) == 0 {
+		return gu, errors.New("url cannot be empty")
+	}
+	if len(hash) != HashLength {
+		return gu, errors.New("hash must be 32 bytes")
```

Comment:
```suggestion
		return gu, fmt.Errorf("hash must be %d bytes", HashLength)
```

Reply:
sha256 is 32 bytes the hexa encoding is 64

Reply:
yes, but the error is misleading. we can do something like `hex encoded sha256 hash must to be 64 bytes`.